package.json

```diff
@@ -49,6 +49,7 @@
     "@babel/eslint-parser": "^7.19.1",
     "glob": "^8.1.0",
     "lodash.template": "^4.5.0",
+    "strip-json-comments": "^5.0.0",
```

Comment:
We can remove this dependency if we can find another (simple) way to partially update `tsconfig.json`.